sdk/messaging/azservicebus/admin/admin_client_subscription.go

```diff
@@ -50,6 +46,9 @@ type SubscriptionProperties struct {
 	// Status is the current status of the queue.
 	Status *EntityStatus
 
+	// AutoDeleteOnIdle is the idle interval after which the subscription is automatically deleted.
+	AutoDeleteOnIdle *time.Duration
```

Comment:
I don't mean to start a naming struggle but to me this sounds like a bool

Reply:
I agree, the name's a bit odd. For better or worse it's this way in the other track 2 SDKs, which are that way because they match the ARM layer.